### PR TITLE
Links and copy

### PR DESCRIPTION
**Pull request by jtuckerk**
updating some minor copy and links changes

### CODE REVIEW

**Comment by jtuckerk** [comment]
waiting on the home page copy for the donation card and an image for this PR

File: client/src/components/About.js [hunk 1]
```diff
@@ -114,28 +114,15 @@ function About() {
...
-        <p className="med-text thicc">Team Leads</p>
-        <>{teamLeadProfiles}</>
+        <p style={{ paddingTop: '1rem' }}> Welcome to Humanity Forward Rapid
+        Response. I’m excited to work with you to build the UBI movement
+        nationwide.  We are going to pressure congress, engage elected officials
+        and rally volunteers to make UBI a reality.</p>
+        <p>Our movement is growing; Our message is resonating.</p>
+        <p>This is bigger than politics; This is about building a human-centered
+        society that works for us!</p>
+        <section className="about-section text-center"> <p
```
Comment (ecaoile) [inline client/src/components/About.js:124]: Looks like the p tag is missing a >

**Review by ecaoile — CHANGES_REQUESTED** [review]
>

**Review by ecaoile — APPROVED** [review]
Nm, I lied. It's fine.

**Pull request merged**